src/main/scala/nl.knaw.dans.easy.bagstore/server/FilesizesServletComponent.scala

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright (C) 2016 DANS - Data Archiving and Networked Services (info@dans.knaw.nl)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package nl.knaw.dans.easy.bagstore.server
+
+import nl.knaw.dans.easy.bagstore._
+import nl.knaw.dans.easy.bagstore.component.BagStoresComponent
+import nl.knaw.dans.lib.error._
+import org.joda.time.DateTime
+import org.scalatra._
+
+import scala.util.Failure
+import scala.util.control.NonFatal
+
+trait FilesizesServletComponent {
+  this: BagStoresComponent =>
+
+  val filesizesServlet: FilesizesServlet
+
+  trait FilesizesServlet extends ScalatraServlet with ServletUtils {
+
+    get("/:uuid/*") {
```

Comment:
can't this be part of the `BagsServlet`, like you did with the `StoresServlet`?

Reply:
Right. I deleted the `FilesizesServletComponent` and moved its content into `BagServletComponent`.